add Russian (Student) layout

Zolax9 opened the pull request:
#573
adds a variant of the Russian layout that is used in GBoard and other keyboards for learners of the Russian language (more info and screenshots in the mentioned issue). please review the commit as I'm not entirely sure if I've implemented it correctly!

many thanks!

E̶D̶I̶T̶:̶ ̶*̶*̶d̶o̶ ̶n̶o̶t̶ ̶m̶e̶r̶g̶e̶ ̶y̶e̶t̶*̶*̶ ̶a̶s̶ ̶t̶h̶e̶r̶e̶ ̶i̶s̶ ̶a̶n̶ ̶i̶s̶s̶u̶e̶ ̶w̶i̶t̶h̶ ̶o̶n̶e̶ ̶o̶f̶ ̶t̶h̶e̶ ̶l̶e̶t̶t̶e̶r̶s̶ ̶i̶n̶ ̶t̶h̶e̶ ̶l̶a̶y̶o̶u̶t̶ ̶b̶e̶i̶n̶g̶ ̶d̶i̶s̶c̶u̶s̶s̶e̶d̶ ̶i̶n̶ ̶t̶h̶e̶ ̶r̶e̶l̶a̶t̶e̶d̶ ̶i̶s̶s̶u̶e̶.̶

EDIT2: the issue has been fixed and the layout name was changed from "Russian (Students)" to "Russian (Student)" as a better translation of the layout's name derived from GBoard

Helium314 commented:
The `subtypeId` must be unique, see [layouts.md](https://github.com/Helium314/HeliBoard/blob/main/layouts.md#adding-new-layouts--languages).

Further, is it necessary to have the popup keys for `е` and `ь`?

Otherwise it should be good.

Zolax9 commented:
> The `subtypeId` must be unique, see [layouts.md](https://github.com/Helium314/HeliBoard/blob/main/layouts.md#adding-new-layouts--languages).

oh, my bad. changed to a unique hex number

> Further, is it necessary to have the popup keys for `е` and `ь`?

I was not aware that they were already pop-up keys for the Russian language. thanks, fixed as well!
